scripts/draft-blog-post.js

```diff
@@ -80,22 +82,24 @@ rimraf.sync(postGlob);
 
 fs.writeFileSync(
```

Comment:
Should we run prettier.format on this file?

Reply:
One second, this file is created when repo linting with stable, and merge when master already released,  so there is a chance this file will fail on lint?

Reply:
There may be a chance. But we use this script when before release. So I'm not sure if the problem is fixed with using `prettier.format` on this script.

Reply:
If we use stable prettier format it, it will fail after merge.
If we use master prettier format it, it will fail when sending PR.

😄

Reply:
Let's keep as it is, for now.